.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lib/tinyusb"]
 	path = lib/tinyusb
-	url = https://github.com/hathach/tinyusb.git
+	url = https://github.com/meshtastic/tinyusb.git
```

Comment:
are you sure you are making PR to this repo and not other fork ?

Reply:
Very good eye. 

I started the PR as a draft to take advantage of the CI tests here but would eventually submit it here. 

We’re running this through a few tests at the moment and will take it out of draft once we have confidence and I will write up the change description. 

Thanks!